### DISCUSSION TITLE
[Fast Data] getting started with the Single View Patch

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#142] [Q&A] [answered]
Hi,
I'm about to create a new single view creator that is dedicated to the single view patch operations.
In the marketplace there are 3 options. The plugins (manual and low code) and the template.
Which is the desired option to getting started?

**Comment by ianFar96** [comment, marked as answer]
Hi Omar,

You should choose the Single View Creator - Low Code plugin since it's the best option to use the sv-patch mechanism. I'm not sure if the other two can really use the sv-patch but if you need me I could look in to it!
